x/rewards/keeper/keeper.go

```diff
@@ -120,6 +121,13 @@ func (k Keeper) BondedTargetFactor(ctx sdk.Context) cosmosMath.LegacyDec {
 		// equivalent to: (maxBonded - bonded) / (maxBonded - minBonded)
 		// 					  + lowFactor * (bonded - minBonded) / (maxBonded - minBonded)
 		min_max_diff := maxBonded.Sub(minBonded)
+		if !min_max_diff.IsPositive() {
```

Comment:
I would rather change this in the validate function than to do this here. Go to x/rewards/types/params.go line 101 and change GT() to GTE()